### DISCUSSION TITLE
Piping output to files in Bash

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#161] [Comments]
# Piping output to files in Bash

This is the online playground of Patrick Kollitsch, David's Neighbour, in Germany produced, digital native living & working on Ko Samui, Thailand.

https://kollitsch.dev/blog/2022/piping-output-to-files-in-bash/#giscus-comments

**Comment by brycewray** [comment]
Nicely done! I’ve now copied my existing `start.sh` shell file — which runs `hugo server` with a bunch of switches and flags I can’t begin to remember each time — to a `logstart.sh` file that also uses this method.
